src/components/Register/register.scss

```diff
@@ -0,0 +1,56 @@
+$primary-color: #00d0a0;
+
+body > div, body > div > div, body > div > div > div.login-form {
+  height: 100%;
+}
+
+.form-container {
+  max-width: 380px;
+}
+
+.form-container p {
+  color: rgba(148, 144, 144, 0.829);
+  font-family: -apple-system, BlinkMacSystemFont, Roboto, sans-serif;
+  font-size: small;
+}
+
+.form-container p a {
+  color: $primary-color;
+}
+
+// .button {
```

Comment:
remove the commented code

Reply:
@jomasim ,

Thank you. I have made the change.
Regards
Steven Ennis